homework-10/main.js

```diff
@@ -0,0 +1,154 @@
+function Car(status) {
+    this.serviceability = 100;
+    this.isRiding = status;
+}
+
+Car.prototype.startRide = function () {
+    if (this.isRiding) {
+        throw 'Error: Car already riding';
+    }
+
+    this.isRiding = true;
+    this.serviceability -= Math.random() * (2.5 - 0.1) + 0.1;

```

Comment:
As for me its more readable 
```suggestion
    this.serviceability -= Math.floor(Math.random() * (2.5 - 0.1) + 0.1);
```